modules/exploits/linux/local/glibc_tunables_priv_esc.rb

```diff
@@ -0,0 +1,175 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  # includes: is_root?
+  include Msf::Post::Linux::Priv
+  # includes: kernel_release
+  include Msf::Post::Linux::Kernel
+  # include: get_sysinfo
+  include Msf::Post::Linux::System
+  # includes writable?, upload_file, upload_and_chmodx, exploit_data, cd
+  include Msf::Post::File
+  # includes register_files_for_cleanup
+  include Msf::Exploit::FileDropper
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  BUILD_IDS = %w[69c048078b6c51fa8744f3d7cff3b0d9369ffd53 3602eac894717d56555552c84fc6b0e4d6a4af72 a99db3715218b641780b04323e4ae5953d68a927 a8daca28288575ffc8c7641d40901b0148958fb1 61ef896a699bb1c2e4e231642b2e1688b2f1a61e 9a9c6aeba5df4178de168e26fe30ddcdab47d374 e7b1e0ff3d359623538f4ae0ac69b3e8db26b674 956d98a11b839e3392fa1b367b1e3fdfc3e662f6]
```

Comment:
You might want to use a search'n'replace with a placeholder in the .py file for those, to avoid duplicating the information.

Reply:
Good call